### ISSUE TITLE
Open empty page when surfing from https

### ISSUE THREAD

**Issue by HarelM** [osmlab/osm-auth#29]
I added oauth using this library to the following site:
Http://israelhiking.osm.org.il/
When clicking on the upper right sad face the code activates this library.
This works fine on mobile devices, however if I surf to the sane site using https protocol, I'm getting an empty page from osm oauth page.
Does anyone else has the same problem?

**Comment by HarelM** [comment]
Not just mobile devices.
The default of this library is http instead of https so the browser has the following error:
```
Blocked loading mixed active content “http://www.openstreetmap.org/oauth/request_token”[Learn More]  osmauth.min.js:2:6919
Blocked loading mixed active content “http://www.openstreetmap.org/api/0.6/user/details”[Learn More]
```
The solution is to put the default address as https instead of http, I think,

**Comment by tmcw** [comment]
Does https://github.com/osmlab/osm-auth/pull/20 fix this?

**Comment by HarelM** [comment]
I don't see a release with this fix, am I right?
But it will probably fix this issue...

**Comment by bhousel** [comment]
> I don't see a release with this fix, am I right?
> But it will probably fix this issue...

Sorry about the delay, I totally forgot to make a release that includes the fix.

I just published v1.0.2 which includes https by default and should hopefully avoid the mixed content issues.  Please comment if you find any issues, and I'll reopen if needed..

**Comment by HarelM** [comment]
Please note that 1.0.2 has braking changes in terms of typescript - Build now fails on 1.0.2 while it didn't fail on 1.0.1.
I suggest to change the version number...
Getting this error now:
`Value of type 'OSMAuthConstructor' is not callable. Did you mean to include 'new'?`
previous usage was without the "new" keyword as far as I can tell...
